06-platform-instruction.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: page
+menutitle: PLATFORM 
+permalink: /platform/
+---
+
+# Provide platform information
+
+Submissions to ReScience come with code. We ask information about the platform that was used
+to run the code, the detail depending on the tools and programming languages used.
+
+This information will be included in the file `code/README.md` of the submission. Consider
+the whole execution pipeline for your results for possible additional tools (e.g. Bash,
+random number generator, etc).
+
+## Version information for Python
+
+For Python, NumPy, SciPy and matplotlib (common Python dependencies), the
+following script will provide detailed information on your platform.
+```py
+from __future__ import print_function
+import sys
+import numpy as np
+import scipy
```

Comment:
Is it a good idea to report the version numbers of all these packages, even if they are not used in the submitted code?